src/odin/enhancedtrippath.cc

```diff
@@ -26,18 +29,7 @@ constexpr int kIsStraightestBuffer = 10;                   // Buffer between str
 
 constexpr uint32_t kBackwardTurnDegreeLowerBound = 124;
 constexpr uint32_t kBackwardTurnDegreeUpperBound = 236;
-
-const std::string& Pronunciation_Alphabet_Name(valhalla::Pronunciation_Alphabet alphabet) {
-  static const std::unordered_map<valhalla::Pronunciation_Alphabet, std::string>
-      values{{valhalla::Pronunciation_Alphabet::Pronunciation_Alphabet_kIpa, "kIpa"},
-             {valhalla::Pronunciation_Alphabet::Pronunciation_Alphabet_kXKatakana, "kXKatakana"},
-             {valhalla::Pronunciation_Alphabet::Pronunciation_Alphabet_kXJeita, "kXJeita"},
-             {valhalla::Pronunciation_Alphabet::Pronunciation_Alphabet_kNtSampa, "kNtSampa"}};
-  auto f = values.find(alphabet);
-  if (f == values.cend())
-    throw std::runtime_error("Missing value in protobuf Pronunciation_Alphabet enum to string");
-  return f->second;
-}
+} // namespace
```

Comment:
this looks like a mistake, you took some things out of the anonymous namespace

Reply:
it's actually on purpose, it's moved to proto_conversions.h as that exact function was created twice in anonymous namespaces (see markup_formatter.cc below)

Reply:
by moving this up to here to pull `RoadClass_Name` `TripLeg_Traversability_Name` `TripLeg_Use_Name` `TripLeg_TravelMode_Name`  `TripLeg_VehicleType_Name` and many more out of the anonymous namespace. totally cool to move that function over to there but the other functions below here should stay in the anonymous namespace

Reply:
ah yeah, I was blind, thanks for fixing @kevinkreiser !